lib/bcoin.js

```diff
@@ -1,7 +1,11 @@
 var bcoin = exports;
 var elliptic = require('elliptic');
 
-bcoin.ecdsa = elliptic.ecdsa(elliptic.nist.secp256k1);
+if (elliptic.ec) {
```

Comment:
Oh, that was for compat with older versions of elliptic. We should probably just upgrade elliptic anyway.